.travis.yml

```diff
@@ -14,22 +32,23 @@ addons:
       - g++-4.9
 
 script:
-  - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.9 90
+  - if [[ "${TRAVIS_OS_NAME}" == "linux" && "${CXX}" == "g++"  ]];
+    then sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.9 90;
+    fi
   - if [[ "${TRAVIS_EVENT_TYPE}" == "cron" && "${CXX}" == "g++"  ]];
     then ./.travis-ci.d/coverity_scan.sh;
     else ./.travis-ci.d/compile_and_test.sh;
     fi
 
 before_install:
   - ./.travis-ci.d/install_dependencies.sh
+  - if [[ "${TRAVIS_OS_NAME}" == "osx"  ]];
+    then brew install mysql;
```

Comment:
No TravisCI addons for that ?

Reply:
Apparently not: https://docs.travis-ci.com/user/reference/osx/#Homebrew

Reply:
Fair !